src/main/io/osd.c

```diff
@@ -252,6 +252,22 @@ static int digitCount(int32_t value)
     return scaled;
 }
 
+/**
+ * Aligns text to the left side. Adds spaces at the end to keep string length unchainged.
+ */
+static void osdLeftAlignString(char *buff)
+{
+  int sp = 0;
```

Comment:
Please use 4 spaces as indentation

Reply:
I think this implementation should be optimised a bit. You could count leading spaces at the same time you advance to check the string length. Also, please use braces even for single statements.